### PR TITLE
Add example programs for adding DOM events listeners

### PR DESCRIPTION
**Pull request by vlad-lubenskyi**
In this pull request, we:
1. Introduce two example programs demonstrating how to listen to the keyboard and mouse DOM events.
2. Apply project-level code style.

### CODE REVIEW

File: dom/src/main/java/KeyboardEventListener.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            frame.add(view, BorderLayout.CENTER);
+            frame.setSize(800, 600);
+            frame.setLocationRelativeTo(null);
+            frame.setVisible(true);
+        });
+        loadHtmlAndWait(browser);
+
+        findInputField(browser)
+                .ifPresent(element -> {
+                    element.addEventListener(EventType.KEY_DOWN, KeyboardEventListener::printEventDetails, false);
```
Comment (vladimir-ikryanov) [inline dom/src/main/java/KeyboardEventListener.java:69]: Reformat. I suppose the line length is more than 80-100 characters.

File: dom/src/main/java/KeyboardEventListener.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+/**
+ * This examples demonstrates how to capture keyboard events from the DOM nodes.
+ */
+public class KeyboardEventListener {
+
+    private static final String INPUT_FIELD_ID = "input-field";
+    private static final String HTML = "<input type='text' id='" + INPUT_FIELD_ID + "' />";
+
+    public static void main(String[] args) {
+        EngineOptions options = EngineOptions.newBuilder(HARDWARE_ACCELERATED)
```
Comment (vladimir-ikryanov) [inline dom/src/main/java/KeyboardEventListener.java:50]: We usually use more compact way:
```java
Engine engine = Engine.newInstance(
        EngineOptions.newBuilder(HARDWARE_ACCELERATED)
                .build());
```

File: dom/src/main/java/MouseEventListener.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+/**
+ * This examples demonstrates how to capture mouse events from the DOM nodes.
+ */
+public class MouseEventListener {
+
+    private static final String BUTTON_ID = "button";
+    private static final String HTML = "<button id='" + BUTTON_ID + "'>Click me</button>";
+
+    public static void main(String[] args) {
+        EngineOptions options = EngineOptions.newBuilder(HARDWARE_ACCELERATED)
```
Comment (vladimir-ikryanov) [inline dom/src/main/java/MouseEventListener.java:51]: Same suggestion here.

File: dom/src/main/java/MouseEventListener.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            frame.add(view, BorderLayout.CENTER);
+            frame.setSize(800, 600);
+            frame.setLocationRelativeTo(null);
+            frame.setVisible(true);
+        });
+        loadHtmlAndWait(browser);
+
+        findButton(browser)
+                .ifPresent(element -> {
+                    element.addEventListener(EventType.MOUSE_DOWN, MouseEventListener::printEventDetails, false);
```
Comment (vladimir-ikryanov) [inline dom/src/main/java/MouseEventListener.java:70]: Same issue with line length.

File: dom/src/main/java/MouseEventListener.java [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    public static void main(String[] args) {
+        EngineOptions options = EngineOptions.newBuilder(HARDWARE_ACCELERATED)
+                                             .build();
+        Engine engine = Engine.newInstance(options);
+        Browser browser = engine.newBrowser();
+
+        SwingUtilities.invokeLater(() -> {
+            BrowserView view = BrowserView.newInstance(browser);
+
+            JFrame frame = new JFrame("DOM Mouse Event Listener ");
```
Comment (vladimir-ikryanov) [inline dom/src/main/java/MouseEventListener.java:59]: Extra space in the title.

**Review by vladimir-ikryanov — CHANGES_REQUESTED** [review]

File: dom/src/main/java/KeyboardEventListener.java (hunk 2 above)
Comment (vlad-lubenskyi) [reply dom/src/main/java/KeyboardEventListener.java:50]: I find it uglier :)

**Review by vladimir-ikryanov — APPROVED** [review]
LGTM

**Pull request merged**